domain/src/main/java/no/difi/meldingsutveksling/domain/sbdh/ScopeType.java

```diff
@@ -2,5 +2,6 @@
 
 public enum ScopeType {
     JournalpostId,
-    ConversationId
+    ConversationId,
+    MessagetypeId
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00115)